webclient/app/src/app/commons/alert/ErrorAlert.jsx

```diff
@@ -0,0 +1,22 @@
+import {
+    Alert,
+    AlertTitle,
+    Collapse,
+} from "@mui/material";
+import { useTranslation } from "react-i18next";
+
+const ErrorAlert = (props) => {
+    const { alert, onClose } = props;
+    const { t } = useTranslation();
+
+    return (
+        <Collapse in={alert.open} onClose={onClose} >
+        <Alert severity="error">
+            <AlertTitle>{t(alert.title)}</AlertTitle>
```

Comment:
Please move translation to parent component

Reply:
done